### ISSUE TITLE
Warn about missing dependent modules on jar import

### ISSUE THREAD

**Issue by tombentley** [ceylon/ceylon-ide-eclipse#1293]
According to https://github.com/ceylon/ceylon-compiler/issues/2052 the  `Export -> Java archive to Module repository` functionality needs to be changed to warn users when the jar they're trying to import depends on missing modules. The work to make an API accessible to the IDE was done in 
https://github.com/ceylon/ceylon-compiler/issues/2070.

**Comment by gavinking** [comment]
Yeah well I tried, and I found the API is AFAICT totally useless for us. So I dunno, this is not for 1.2, AFAICS.
